plugins/inputs/aerospike/aerospike.go

```diff
@@ -65,67 +72,231 @@ func (a *Aerospike) gatherServer(hostport string, acc telegraf.Accumulator) erro
 		iport = 3000
 	}
 
-	c, err := as.NewClient(host, iport)
+	var c *as.Client
+	if urlInfo.User != nil {
+		userInfo := urlInfo.User
+		clientPolicy := as.NewClientPolicy()
+		clientPolicy.User = userInfo.Username()
+		clientPolicy.Password, _ = userInfo.Password()
```

Comment:
no, just use the tls config for all servers specified in one plugin. If users need multiple tls configs they can specify multiple [[inputs.aerospike]] instances